packages/frontend/src/components/markdown-split-editor/markdown-split-editor.tsx

```diff
@@ -110,23 +110,27 @@ const FormattingHelp = () => (
 );
 
 interface Props {
+  draftKey?: string;
   baseAssetUrl?: string;
   baseLinkUrl?: string;
   contents: string;
   onChange: (updatedContents: string) => void;
   showFormattingHelp?: boolean;
   showPreview?: boolean;
   transformAssetUri?: ((uri: string, children?: ReactNode, title?: string, alt?: string) => string) | null;
+  uploadFile?: (file: any, name: string) => any;
```

Comment:
Once you firm up the `uploadFile` type (see above), can you update this and the other locations to match?